src/options.js

```diff
@@ -217,7 +217,17 @@ export function maybeColumn(source) {
 }
 
 export function labelof(value, defaultValue) {
-  return typeof value === "string" ? value : value && value.label !== undefined ? value.label : defaultValue;
+  return typeof value === "string"
+    ? value
+    : typeof value === "function" && value.name && !/^f?[xy]\d*$/.test(value.name)
+    ? value.name
+    : value && value.label !== undefined
+    ? value.label
+    : defaultValue;
+}
+
+export function anonymous(f) {
```

Comment:
This is a trick to avoid _e.g._ `const bin = (d) => …` inheriting the name `bin`.